discovery-infra/tests/test_cancel_and_reset.py

```diff
@@ -233,6 +233,7 @@ def test_cancel_reset_after_installation_failure(self, nodes, cluster):
         # new_cluster.start_install_and_wait_for_installed()
 
     @pytest.mark.regression
+    @pytest.mark.wrong_boot_order
```

Comment:
what is the idea of this one? it is added for all wrong_boot_order tests?

Reply:
i think i got it, u mean it is testing 2 things , cancel reset and wrong boot order?

Reply:
First reason is that there ia another wrong_boot_order test that is under cancel\reset. 
the second reason is that wrong boot order has 1 test, added by Raz, that validate that there is no timeout when cluster is in this state (timeout removed) so we wait for 1 h. I understood from Raz that in the future the timeout will be changed to 24 h, so i would like us to have the ability to run wrong_boot_order seperatly